recipes/nb_black/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "nb_black" %}
+{% set version = "1.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 33247d55b7bfa8d1d0725c7ba0dbd1984b386ef790d5bd1f7eb7c8826cdcfea5
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - black
```

I know it’s implicit from the other two but it’s probably nicer to include `python` explicitly:
```suggestion
    - python
    - black
```